### PR TITLE
 Born of .travis.yml

### PR DESCRIPTION
**Pull request by baevpetr**
Initialization of Travis-CI.
@annkupriyanova have to make following steps:
- Go to [Travis-ci.com](url) and Sign up with GitHub.
- Accept the Authorization of Travis CI. You’ll be redirected to GitHub.
- Click the green Activate button, and select the repositories you want to use with Travis CI.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dist: xenial      # 12.04 Precise default
```
Comment (cdeler) [inline .travis.yml:1]: Why not 18.04?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+language: python
+python:
+  - "3.6"         # 2.7 default
+cache: pip
+install:
+  - pip install -r requirements.txt
+script:
+   pwd #- python -m pytest -v
```
Comment (cdeler) [inline .travis.yml]: Please remove unused comments

+ why did you comment the test execution out?

**Review by cdeler — CHANGES_REQUESTED** [review]

**Comment by cdeler** [comment]
Also good idea (for me) is to squash all commits with minor changes into the single one

File: .travis.yml (hunk 1 above)
Comment (baevpetr) [reply .travis.yml:1]: The reason is that Travis doesn't support 18 version. I assume that this is a question with a hint. Then there's an opportunity to use 18.04 with Docker.

**Comment by baevpetr** [comment]
My fault. Forgot to use `--amend`...

File: .travis.yml (hunk 2 above)
Comment (baevpetr) [reply .travis.yml]: Must be `script`. But there are no tests) I will fix.

**Review by annkupriyanova — COMMENTED** [review]
I've fulfilled the instruction and activated Travis CI for this repository. Check it.

**Comment by baevpetr** [comment]
I can't see your account. So approve PR if it ok for you. And I'm just waiting further commands\clarifications from @cdeler about version of Ubuntu and usage of Docker.

**Review by cdeler — APPROVED** [review]

**Comment by cdeler** [comment]
Just resolve the conflict :-)

**Review by annkupriyanova — APPROVED** [review]

**Pull request closed without merging**